Commonalities/documentation/CAMARA-AuthN-AuthZ-Concept.md

```diff
@@ -141,6 +148,8 @@ The authorization code is a temporary code that the client will exchange for an
 
 Proof Key for Code Exchange (PKCE is specified in RFC 7636) is a kind of proof of possession. It is an extension to the authorization code flow to prevent CSRF and authorization code injection attacks. The technique involves the client first creating a secret on each authorization request, and then using that secret again when exchanging the authorization code for an access token. This way if the code is intercepted, it will not be useful since the token request relies on the initial secret. PKCE uses cryptography to guarantee that the client exchanging an OAuth2 code for tokens is the same client that started the original OAuth2 request. It is hence important to use this extension when using Auth code grant flow.
 
+#### OIDC Client-Initiated Backchannel Authentication (CIBA) flow
+CIBA is an authentication flow like regular OpenID Connect. However, unlike OpenID Connect, there is direct Relying Party to OpenID Provider communication without redirects through the user's browser. It decouples the concept of a Consumption Device (on which the user interacts with the Relying Party) and an Authentication Device (on which the user authenticates with the OpenID Provider and grants consent). CIBA allows a Relying Party that has an identifier for a user to obtain tokens from the OpenID Provider. The user starts the flow with the Relying Party at the Consumption Device, but authenticates and grants consent on the Authentication Device. Authentication could even be initiated without user action on the consumption device.
```

Comment:
It would be great if we could use the line from official docs, regarding authentication device. 
Instead of "but authenticates and grants consent on the Authentication Device", I would suggest "but authenticates and authorizes request on the Authentication Device"



Client-Initiated Backchannel Authentication (CIBA) is an authentication flow in which the Relying Party, that can obtain a valid identifier for the user they want to authenticate, will be able to initiate an interaction flow to authenticate their users without having end-user interaction from the consumption device. The flow involves direct communication from the Client to the OpenID Provider without redirect through the user's browser (consumption device).

Reply:
```suggestion
CIBA is an authentication flow like regular OpenID Connect. However, unlike OpenID Connect, there is direct Relying Party to OpenID Provider communication without redirects through the user's browser. It decouples the concept of a Consumption Device (on which the user interacts with the Relying Party) and an Authentication Device (on which the user authenticates with the OpenID Provider and grants consent). CIBA allows a Relying Party that has an identifier for a user to obtain tokens from the OpenID Provider. The user starts the flow with the Relying Party at the Consumption Device, but authenticates and authorizes request on the Authentication Device. Authentication could even be initiated without user action on the consumption device.
```
@shilpa-padgaonkar Regarding the additional line you suggested, actually on what refers to CIBA, it is a backchannel auth.  So it would be Marketplace (or app BE if there is no Marketplace in between) the one initiating the flow against OpenID Provider (Auth Server) providing the login_hint_token. 

**I still don't have contributor permission, so if you are happy with the change suggested you may apply it :)**

Reply:
@jpengar : I am fine with the suggestion, but would keep the wording similar for the authentication device.

"It decouples the concept of a Consumption Device (on which the user interacts with the Relying Party) and an Authentication Device (on which the user authenticates with the OpenID Provider and authorizes the request)"

If you are ok, I will commit this change

Reply:
@jpengar : Regarding additional line, I understand that the relying party here would be marketplace/app-backend and it initiates the flow. 
CIBA doc even mentions it clearly "this is a use case where the RP needs to be the initiator of the user authentication flow, and end-user interaction from the consumption device is not needed." and then it adds the above mentioned line as well that as the user does not provide authentication credentials directly to the consumption device, supporting this flow requires the OpenID Provider to have some mechanism of initiating user authentication out-of-band from the interaction with the consumption device."

Reply:
;) no more change.. commit done